packages/sanity/src/core/config/prepareConfig.ts

```diff
@@ -629,7 +628,8 @@ function resolveSource({
     },
     beta: {
       treeArrayEditing: {
-        enabled: arrayEditingReducer({config, initialValue: false}),
+        // This beta feature is no longer available.
+        enabled: false,
```

Comment:
Just to double check, do we have a plan to go through and clean up the code (since we are using `@deprecated` across) or for now it's just about making the code inaccessible?